perftracker/models/job.py

```diff
@@ -45,6 +45,7 @@ class JobModel(models.Model):
 
     project         = models.ForeignKey(ProjectModel, help_text="Job project", on_delete=models.CASCADE)
 
+    regression      = models.ForeignKey('perftracker.RegressionModel', help_text="Regression of this job", related_name="regr", null=True, blank=True, on_delete=models.CASCADE)
```

Comment:
Now it's not clear what is the difference between:
- regression
and
- regression_tag

one of the attributes is what was assigned on test start and the other is what user manually included/excluded on th portal.

What is what?

I'd propose to have it like this:

1. regression_tag - is an immutable attribute set by the test client. Technically there can be a tag to a removed regression.

2. regression - is a foreign key to the ModelRegression object, i.e. this is real link. This is managed by the user of the perftracker portal